Makefile

```diff
@@ -12,21 +12,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-NAME=system-agent
+NAME=aospace
 EXE=$(NAME)$(shell go env GOEXE)
 DATE=$(shell date '+%F-%T')
-VERSION=$(NAME)-AO.space-$(DATE)
-VERSIONNUMBER=1.0.0
+BRANCH=$(shell git symbolic-ref HEAD | cut -d"/" -f 3)
+COMMIT=$(shell git rev-parse HEAD | cut -c1-7)
+# ARCH ?= arm64
```

Comment:
@zxln007 If the code will no longer be used, it would be better to remove it entirely rather than leaving it as a comment.